test/marketplace/MarketPlace.sol

```diff
@@ -39,14 +38,14 @@ contract MarketPlace {
   }
 
   /// @notice Allows the admin to transfer ownership
-  /// @param t
-  function transferAdmin(address t) external onlyAdmin(admin) returns (bool) {
+  /// @param t Address admin is being transferred to 
+  function transferAdmin(address t) external onlyAddress(admin) returns (bool) {
```

Comment:
oh, so that it can be delegated to another address? we aint gunna use `onlyAddress` tho..

why not just leave it? what's wrong with calling the authorized address admin? ... ic... you want to normalize it across contracts, ok...

just use `authorized` then. we can drop the `only`

Reply:
The ticket essentially pointed out that we have both onlyAdmin and onlySwivel, and that we could just have one that uses the address as a param, instead of seperate modifiers (saving bytecode)

Reply:
done